docs/input_file_formats.md

```diff
@@ -4,60 +4,46 @@
 ### Donor variant and donor alignment list files
 The donor variant list file should contain a list of paths to a variant, VCF or BCF, file with one path per file line.
 The donor alignment list file should be similar and contain one path to a donor alignment file per file line.
 The files listed will be used as donor variant and alignments files respectively when constructing variant contexts.
 Only paths to the variant/alignment files should be provided, not the index files. These are assumed to be in the same 
 directory with the same name, i.e.: sample.bam & sample.bam.bai
-<br /><br />
 
 ### Genome reference
 As VaSeBuilder can work with both BAM and CRAM alignment files, users need to supply a genome reference in fasta format for each VaSeBuilder run. This genome reference should be the same as the reference used during the mapping process to create the acceptor and donor alignment files.
 
-
 ### Donor FastQ list file
-A donor fastq list file needs to be provided when running VaSeBuilder AC-mode. Currently (September 23rd, 2019) this 
-file requires two columns without a header. The first column should contain the paths to R1 files, the second column
- the paths to R2 files:
+_Note: Using VaSeBuilder-produced FastQ spike-in building blocks is deprecated in favor of using BAMs, as this allows for more flexibility and provides more information._
 
-```
-/path/to/sample1_R1.fq &emsp;/path/to/sample1_R2.fq
-/path/to/sample2_R1.fq &emsp;/path/to/sample2_R2.fq
-/path/to/sample3_R1.fq &emsp;/path/to/sample3_R2.fq
-```
-
-Each row in this file therefore represents a single sample.
-<br /><br />
+A donor fastq list file can be provided when running `AssembleValidationSet`. This file requires two columns without a header. The first column should contain the paths to R1 files, the second column the paths to their R2 counter-parts:
 
+```text
+/path/to/sample1_R1.fq	/path/to/sample1_R2.fq
+/path/to/sample2_R1.fq	/path/to/sample2_R2.fq
+/path/to/sample3_R1.fq	/path/to/sample3_R2.fq
+```
 
 ### Variant list file
-A variant list file can be provided as a filter to indicate which donor variants to use in the VaSeBuilder run. This is 
-most useful when only certain variants from the donors are required but the donor variant files contain many variants.
+A variant list file can be provided as a filter to indicate which donor variants from which samples to use in the VaSeBuilder run. This is most useful when only certain variants from the donors are required, but the donor variant files contain many variants.
 Variants not specified in the variant list will therefore be skipped.
-The variant list file needs at least three columns: sample name/identifier, chromosome name, variant position:
+The variant list file requires the following 5 columns: sample name/identifier, chromosome name, 1-indexed variant start position, reference allele, and alt allele, with the column headers shown below, and can also have additional user-defined custom columns:
 
+```text
```

Comment:
I'm a little torn about these example text files since they don't columnize well in markdown. It makes it a little hard to read. As an alternative, we could do table formatting. This makes it much clearer, but it also makes the example less literal:

Sample|Chrom|Pos|Ref|Alt|Pathogenicity
:--|:--|:--|:--|:--|:--
sample1|3|12345678|A|T|benign
sample2|4|1357911|GCC|G|likely_pathogenic
sample3|7|24681012|T|TTT|benign